keyboards/maxr1998/pulse4k/config.h

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Maxr1998
+#define PRODUCT         Pulse 4k
+#define DESCRIPTION     A four-key macropad
+
+/* Key matrix size */
+#define MATRIX_ROWS 2
+#define MATRIX_COLS 3
+
+/* Matrix pins */
+#define MATRIX_ROW_PINS { B4, E6 }
+#define MATRIX_COL_PINS { B7, B3, F0 }
+#define UNUSED_PINS
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCING_DELAY 5
```

Comment:
```suggestion
#define DEBOUNCE 5
```
This definition has been renamed

Reply:
Alright, will fix. Not sure from where I copied this error :thinking:

Reply:
A lot of the config.h files in the repo still use `DEBOUNCING_DELAY`, and it was only recently changed, so no biggie.

Reply:
I see, thanks!